CS483/CS483/Kartaclysm/Components/ComponentTrack.cpp

```diff
@@ -466,6 +474,19 @@ namespace Kartaclysm
 		}
 	}
 
+	bool ComponentTrack::IsUnderJump(int p_iTrackPiece)
+	{
+		for (int i = 0; i < m_vUnderTrackPieces.size(); i++)
+		{
+			if (p_iTrackPiece == m_vUnderTrackPieces[i])
+			{
+				return true;
+			}
```

Comment:
Not that this is bad, but couldn't this be summed up in one line use std::find() or accessing the boolean on the track piece directly. This will also avoid a signed/unsigned mismatch warning